### PR TITLE
Encode embedded files as []byte slices

### PR DESCRIPTION
**Pull request by a-palchikov**
This is the tool I use to embed the SELinux policy files inside the binary.
I stripped away most of the complexity (incl. the implicit use of compression which I don't need).

The choice of []byte slice vs the existing string literal is pragmatic - the string literal was causing my diff viewer to fail to compute a diff (probably due to a memory consumption issues).

This tool is a temporary means which might be replaced by something else (read: better) - perhaps once [this](https://github.com/golang/go/issues/35950) has been settled on and implemented.

### CODE REVIEW

File: bytewriter.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+// byteWriter encodes the input as a []byte literal.
+type byteWriter struct {
+	w     io.Writer
+	ntabs int
+	err   error
+}
+
+func (r *byteWriter) Write(p []byte) (n int, err error) {
+	for len(p) > 0 {
+		s := 16
```
Comment (r0mant) [inline bytewriter.go]: I would attempt to make this function more comprehensible for an average reader (like myself 😄). I.e. what is "s"? Why is it equal 16? What is indent and why is it needed? And comments before each "if" branch would be helpful too.

File: bytewriter.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+			s = len(p)
+		}
+		r.indent()
+		r.write(p[0])
+		if s > 1 {
+			for _, c := range p[1:s] {
+				r.writeAdditional(c)
+			}
+		}
+		if len(p) > 0 && s == 16 {
```
Comment (r0mant) [inline bytewriter.go]: Isn't "p" already >0 due to the check above?

File: bytewriter.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+		p = p[s:]
+	}
+	return n, r.err
+}
+
+func (r *byteWriter) indent() error {
+	if r.err != nil {
+		return r.err
+	}
+	_, r.err = fmt.Fprint(r.w, strings.Repeat("\t", r.ntabs))
```
Comment (r0mant) [inline bytewriter.go:47]: Going to my comment above - would be nice to explain why indent is needed.

File: bytewriter.go [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+func (r *byteWriter) writeAdditional(c byte) error {
+	if r.err != nil {
+		return r.err
+	}
+	_, r.err = fmt.Fprintf(r.w, " 0x%02x,", c)
+	return r.err
+}
+
+func (r *byteWriter) writeEol() error {
```
Comment (r0mant) [inline bytewriter.go]: Nit: EOL?

File: bytewriter.go (hunk 3 above)
Comment (a-palchikov) [reply bytewriter.go:47]: Added the comment on the struct - it outputs the byte slices adhering to formatting rules as `gofmt` would format w/o requiring `gofmt`.

**Review by knisbet — APPROVED** [review]

**Pull request merged**